packages/mask/shared-ui/locales/en-US.json

```
@@ -1062,6 +1062,19 @@
     "nft_wallet_change": "Change",
     "nft_button_add_collectible": "Add Collectible",
     "nft_avatar": "NFT Avatar",
+    "plugin_ideamarket_smthg_wrong": "Failed to load, please retry later.",
```

Don't use uncommon abbreviation, also don't forget to remove `smthg` in `cspell.json`.
```suggestion
    "plugin_ideamarket_something_wrong": "Failed to load, please retry later.",
```

```suggestion
    "plugin_ideamarket_sth_wrong": "Failed to load, please retry later.",
```

---

Done.